autoextract/aio/client.py

```diff
@@ -14,8 +15,10 @@
 from autoextract.utils import chunks, user_agent
 from autoextract.request import Query, query_as_dict_list
 from autoextract.stats import ResponseStats, AggStats
-from .retry import autoextract_retry
-from .errors import ApiError
+from .retry import autoextract_retry, QueryRetryError
+from .errors import RequestError, QueryError
+
+logger = logging.getLogger('autoextract')
```

Comment:
Why do you prefer explicit logger name instead of `logger = logging.getLogger(__name__)` here?

Reply:
Not a preference. I've probably copied this line from `__main__.py`. 
Changing to `__name__`.